app/models/concerns/media_youtube_profile.rb

```diff
@@ -90,8 +90,8 @@ def data_from_youtube_profile
 
   def get_youtube_username
     match = self.url.match(/^https?:\/\/(www\.)?youtube\.com\/user\/([^\/]+)/)
-    username = match.nil? ? self.data['title'].gsub(/[^a-zA-Z0-9]/, '') : match[2]
-    username
+    username = match.nil? ? self.data[:raw][:api][:title].gsub(/[^a-zA-Z0-9]/, '') : match[2]
```

Comment:
`self.data['title']` is equal to `self.data[:raw][:api][:title]` (as per https://github.com/meedan/pender/blob/develop/app/models/concerns/media_youtube_profile.rb#L30). What's different here?

Reply:
Afaict, the difference is that the function is called before `data[title]` is set.

Reply:
ok